### ISSUE TITLE
Passing invalid value in sync-async header

### ISSUE THREAD

**Issue by sonalnegivarma** [nhsconnect/integration-adaptor-mhs#12]
Description: This is a negative test. It will verify Asynchronous Reliable response with passing invalid sync-async in header value.
Test Data: 
Content-Type: application/json
Interaction-Id : REPC_IN150016UK05
sync-async    :  invalid_syncasync_123
from-asid      :  918999198938

Expected result: Either 400 bad request or 500 internal server error
Actual Result-   202 (ACCEPTED)

**Comment by michael-kainos** [comment]
@sonalnegivarma please confirm this is fixed in the latest release

**Comment by sonalnegivarma** [comment]
@michael-kainos  I have tested the defect in the latest release. Issue is fixed now.
Test Data:
Content-Type: application/json
Interaction-Id : REPC_IN150016UK05
sync-async : invalid_syncasync_123
from-asid : 918999198942

Expected result: 400 Bad request.
Actual Result- 400 Bad request

Additional details with screenshot has been added to https://medicaldirector.atlassian.net/browse/NHST-98

Thanks!
